source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -256,17 +256,21 @@ This sets the memory size for the event correlation engine.
 white_list
 ^^^^^^^^^^
 
-This specifies an IP for which Active Responses will not be triggered. Only one IP may be specified for each ``<while_list>`` tag, but several IPs may be used by including multiple ``<white_list>`` tags.
+This specifies an IP, netblock, or hostname for which Active Responses will not be triggered. Only one of those values may
+be specified for each ``<while_list>`` tag, but several values may be used by including multiple ``<white_list>`` tags.
+For this to work the field **srcip** needs to be extracted from the event in the correspondent decoder as it is the one that
+will be use to compare against the values defined in white_list.
 
-+--------------------+----------------------------+
-| **Default value**  | n/a                        |
-+--------------------+----------------------------+
-| **Allowed values** | Any IP address or netblock |
-+--------------------+----------------------------+
++--------------------+-----------------------------------------------------------+
+| **Default value**  | n/a                                                       |
++--------------------+-----------------------------------------------------------+
+| **Allowed values** | Any IP address, netblock (Ej: 192.168.0.0/16) or hostname |
```

Comment:
```suggestion
| **Allowed values** | Any IPv4 address, IPv4-mapped IPv6 address, netblock (i.e.: 192.168.0.0/16) or hostname |
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)